src/cctp-sdk/index.ts

```diff
@@ -32,112 +32,129 @@ interface AttestationStatus {
   attestation: string;
 }
 
+interface Authorization {
+  validAfter: number;
+  validBefore: number;
+  nonce: Uint8Array;
+  v: number;
+  r: string;
+  s: string;
+}
+
 export const transfer = async (
```

Comment:
split this up into separate `relayDeposit` and `relayReceive` and then the combined `relayDepositAndReceive`